package/MDAnalysis/coordinates/PDB.py

```diff
@@ -217,6 +216,17 @@ class PDBReader(base.ReaderBase):
     79 - 80        LString(2)    charge       Charge  on the atom.
     =============  ============  ===========  =============================================
 
+    Notes
+    -----
+    If a system does not have unit cell parameters (such as in EM structures),
+    the PDB file format requires the CRYST1 field to be provided with unitary
+    values (cubic box with sides of 1Å) and an appropriate REMARK. If unitary
+    values are found within the CRYST1 field, :code:`PDBReader` will not set
+    unit cell dimensions and it
+    will warn the user.
+
+    .. _CRYST1: http://www.wwpdb.org/documentation/file-format-content/format33/sect8.html#CRYST1
```

Comment:
Does sphinx complain about a duplicated link between here and further down? If so, keep only 1.